airflow/providers/google/cloud/log/gcs_task_handler.py

```diff
@@ -144,28 +174,15 @@ def gcs_write(self, log, remote_log_location):
         :type remote_log_location: str (path)
         """
         try:
-            old_log = self.gcs_read(remote_log_location)
+            blob = storage.Blob.from_string(remote_log_location, self.client)
+            old_log = blob.download_as_string()
             log = '\n'.join([old_log, log]) if old_log else log
         except Exception as e:  # pylint: disable=broad-except
```

Comment:
nit: btw. isn't it worth to log the exeption?

Reply:
I wrote a message to the log  Thanks.